Package.swift

```diff
@@ -107,7 +107,7 @@ let package = Package(
 
     .target(
       name: "SwiftIDEUtils",
-      dependencies: ["SwiftSyntax", "SwiftParser"],
+      dependencies: ["SwiftSyntax", "SwiftSyntaxMacroExpansion", "SwiftDiagnostics", "SwiftParser"],
```

Comment:
Is `SwiftSyntaxMacroExpansion` left over from when it was moved there?

Reply:
Good catch. Removed the dependency.